app/screens/Info.js

```diff
@@ -1,17 +1,43 @@
 import React from 'react'
 import { SafeAreaView, StyleSheet, ScrollView, View } from 'react-native'
 import { colors } from 'app/config'
+import ListHeader from '../components/common/ListHeader'
 import ListItem from '../components/common/ListItem'
+import { useSelector } from 'react-redux'
 
-const Info = ({ navigation }) => {
+const Info = ({ navigation, route }) => {
+  const rackingData = useSelector((state) => state.rackings.data)
+  const { rackings, systems } = rackingData
+  const { systemId, rackingChildren } = route.params
+  const system = systems.find((syst) => {
+    return syst.id === systemId
```

Comment:
You could create a selector to select items by id. Gives you the benefit of reusing the function easily across components without rewriting it.